Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -146,23 +136,34 @@ - (UIView *)preferredFocusedView
 }
 #endif
 
+- (BOOL)canBecomeFirstResponder
+{
+  ASDisplayNodeAssertMainThread();
+  return [self __canBecomeFirstResponder];
+}
+
+- (BOOL)canResignFirstResponder
+{
+  ASDisplayNodeAssertMainThread();
+  return [self __canResignFirstResponder];
+}
+
 - (BOOL)isFirstResponder
 {
   ASDisplayNodeAssertMainThread();
-  return _view != nil && [_view isFirstResponder];
+  return [self __isFirstResponder];
 }
 
-// Note: this implicitly loads the view if it hasn't been loaded yet.
```

Comment:
Seems like a good comment to keep.

Reply:
We have this kind of comment in `__becomeFirstResponder` now:
```
// Note: this implicitly loads the view if it hasn't been loaded yet.
```

Let me know if I should also add it to this place back again.